doc/user_guide/user_guide.rst

```diff
@@ -0,0 +1,218 @@
+Script-Languages-Container-Tool User Guide
```

Comment:
minor thing; can ignore; out of scope feasible.

User guide uses 'This is my title format' and developer guide uses "This is my Title Format"